commit cd5897bdda84388e1b859847b2263709bb254860
Author: Rob Hoes <rob.hoes@citrix.com>
Date:   Wed Nov 6 17:49:51 2013 +0000

    libxl: ocaml: add domain_build/create_info/config and events to the bindings.
    
    We now have enough infrastructure in place to do this trivially.
    
    Signed-off-by: Ian Campbell <ian.campbell@citrix.com>
    Signed-off-by: Rob Hoes <rob.hoes@citrix.com>
    Acked-by: Ian Jackson <ian.jackson@eu.citrix.com>
    Acked-by: David Scott <dave.scott@eu.citrix.com>

diff --git a/tools/libxl/libxl_types.idl b/tools/libxl/libxl_types.idl
index c43c5b1..ce003c6 100644
--- a/tools/libxl/libxl_types.idl
+++ b/tools/libxl/libxl_types.idl
@@ -464,7 +464,7 @@ libxl_domain_config = Struct("domain_config", [
     ("on_reboot", libxl_action_on_shutdown),
     ("on_watchdog", libxl_action_on_shutdown),
     ("on_crash", libxl_action_on_shutdown),
-    ])
+    ], dir=DIR_IN)
 
 libxl_diskinfo = Struct("diskinfo", [
     ("backend", string),
